x-pack/test/security_solution_cypress/cypress/tasks/prebuilt_rules.ts

```diff
@@ -35,190 +29,93 @@ export const ruleUpdatesTabClick = () => {
   cy.location('pathname').should('include', RULES_UPDATES);
 };
 
-interface RuleInstallUpgradeAssertionPayload {
-  rules: Array<typeof SAMPLE_PREBUILT_RULE>;
-  didRequestFail?: boolean;
-}
-
-export const assertRuleAvailableForInstallAndInstallOne = ({
-  rules,
-  didRequestFail = false,
-}: RuleInstallUpgradeAssertionPayload) => {
-  interceptInstallationRequestToFail(rules, didRequestFail);
-  const rule = rules[0];
-  cy.get(getInstallSingleRuleButtonByRuleId(rule['security-rule'].rule_id)).click();
-  cy.wait('@installPrebuiltRules');
-  assertInstallationSuccessOrFailure([rule], didRequestFail);
-};
-
-export const assertRuleAvailableForInstallAndInstallSelected = ({
-  rules,
-  didRequestFail = false,
-}: RuleInstallUpgradeAssertionPayload) => {
-  interceptInstallationRequestToFail(rules, didRequestFail);
-  let i = 0;
+export const assertInstallationRequestIsComplete = (rules: Array<typeof SAMPLE_PREBUILT_RULE>) => {
   for (const rule of rules) {
-    cy.get(RULE_CHECKBOX).eq(i).click();
-    cy.get(ADD_ELASTIC_RULES_TABLE).contains(rule['security-rule'].name);
-    i++;
+    cy.get(getInstallSingleRuleLoadingSpinnerByRuleId(rule['security-rule'].rule_id)).should(
+      'exist'
+    );
   }
-  cy.get(INSTALL_SELECTED_RULES_BUTTON).click();
-  cy.wait('@installPrebuiltRules');
-  assertInstallationSuccessOrFailure(rules, didRequestFail);
-};
-
-export const assertRuleAvailableForInstallAndInstallAllInPage = ({
-  rules,
-  didRequestFail = false,
-}: RuleInstallUpgradeAssertionPayload) => {
-  interceptInstallationRequestToFail(rules, didRequestFail);
   for (const rule of rules) {
-    cy.get(ADD_ELASTIC_RULES_TABLE).contains(rule['security-rule'].name);
+    cy.get(getInstallSingleRuleLoadingSpinnerByRuleId(rule['security-rule'].rule_id)).should(
+      'not.exist'
+    );
   }
-  cy.get(SELECT_ALL_RULES_ON_PAGE_CHECKBOX).click();
-  cy.get(INSTALL_SELECTED_RULES_BUTTON).click();
-  cy.wait('@installPrebuiltRules');
-  assertInstallationSuccessOrFailure(rules, didRequestFail);
 };
 
-export const assertRuleAvailableForInstallAndInstallAll = ({
-  rules,
-  didRequestFail = false,
-}: RuleInstallUpgradeAssertionPayload) => {
-  interceptInstallationRequestToFail(rules, didRequestFail);
+export const assertUpgradeRequestIsComplete = (rules: Array<typeof SAMPLE_PREBUILT_RULE>) => {
   for (const rule of rules) {
-    cy.get(ADD_ELASTIC_RULES_TABLE).contains(rule['security-rule'].name);
+    cy.get(getUpgradeSingleRuleLoadingSpinnerByRuleId(rule['security-rule'].rule_id)).should(
+      'exist'
+    );
+  }
+  for (const rule of rules) {
+    cy.get(getInstallSingleRuleLoadingSpinnerByRuleId(rule['security-rule'].rule_id)).should(
```

Comment:
Looks like a typo, we need to check the "upgrade" spinner here

Reply:
Thanks for catching this! Fixing